src/tests/ftest/util/command_utils.py

```diff
@@ -77,6 +78,8 @@ def __str__(self):
         value = super().__str__()
         if self.sudo:
             value = " ".join(["sudo -n", value])
+        if self.sudo_E:
+            value = " ".join(["sudo -n -E", value])
```

Comment:
If the above comment is applied this would be removed
```suggestion
```
and lines 79-80 would look like this:
```
        if self.sudo_args is not None:
            value = " ".join(["sudo", self.sudo_args, value])
```

Reply:
3e2406951699031108a90fc15df68a95f661d349